src/components/Button/Button.tsx

```diff
@@ -0,0 +1,154 @@
+import React from 'react';
+import {
+  ActivityIndicator,
+  StyleSheet,
+  Text,
+  TouchableOpacity,
+} from 'react-native';
+import {ButtonProps} from './ButtonProps.types';
+import {scale, verticalScale} from '../../utils';
+import {useThemeColor} from '../../hooks';
+import Icon from 'react-native-vector-icons/Ionicons';
+
+export const Button = (props: ButtonProps) => {
+  const {
+    children,
+    mode,
+    isLoading,
+    style,
+    textStyle,
+    disabled,
+    onPress,
+    icon,
+    iconRight,
+    iconStyle,
+    loaderColor,
```

Comment:
Maybe we can give a default value for this?

Reply:
Fixed. I have added a default color of colors.loader